src/styles/card.scss

```diff
@@ -0,0 +1,117 @@
+@import './variables/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $light-gray;
+  border-radius: 5px;
+  margin: 16px;
+  height: auto;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+
+    .image {
+      background-image: url(../images/imac.jpeg);
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    line-height: 18px;
+    height: 36px;
+    width: 166px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $secondary-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    padding: 0 16px;
+    width: 100%;
+    justify-content: center;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $gray;
+  }
+
+  &__reviews {
+    display: flex;
+    margin: 16px;
+    margin-bottom: 0;
+    width: 166px;
+    justify-content: space-between;
+
+    .reviews {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 16px;
+      color: $secondary-color;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    display: flex;
+    width: 166px;
+    line-height: 18px;
+    justify-content: space-between;
+    margin: 24px 16px 0;
+
+    .price {
+      font-size: 12px;
+      font-weight: 400;
+      color: $gray;
+    }
+
+    .amount {
+      font-size: 16px;
+      font-weight: 700;
+      color: $secondary-color;
+    }
+  }
+
+  .button {
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    width: 166px;
+    height: 40px;
+    margin: 16px;
+    border-radius: 5px;
+    background: $primary-color;
+    transition:
+      background 0.3s,
+      border 0.3s;
+
+    &__buy {
+      box-sizing: border-box;
+      text-decoration: none;
+      text-transform: uppercase;
+      color: white;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+    }
+
+    &:hover {
+      box-sizing: border-box;
+      background: white;
+      border: 1px solid $primary-color;
```

Comment:
The hover effect changes the background to white and adds a border. Ensure that this contrast is sufficient for accessibility and that the text remains readable.